src/backends.jl

```diff
@@ -135,7 +135,7 @@ function pickDefaultBackend()
             Pkg.installed(env_default)  # this will error if not installed
             sym = Symbol(lowercase(env_default))
             if haskey(_backendType, sym)
-                return backend(sym)
+                return @eval backend($sym)
```

Comment:
This causes the warning `You have set PLOTS_DEFAULT_BACKEND=GR but it is not installed.` for me, because `$sym` evals to `Plots.gr` instead of `:gr` here and `backend(Plots.gr)` throws a `MethodError`.

Besides, these `@eval`s in `pickDefaultBackend()` don't seem to solve the world age issue for GR. I don't know about the other backends.

Solutions for GR (and probably analogously for other backends) I found to work would be either
* to replace each function `GR.f(args...)` in `src/backends/gr.jl` with `Base.invokelatest(GR.f, args...)` (no idea if this has an effect on performance - it definitely worsens readability),
* to add `backend()` in `@init` in `src/Plots.jl` (This however throws the warning `WARNING: using Plots.GR in module Main conflicts with an existing identifier.`) or
* to expect from the users to call `backend()` or specify their backend before the first plot.